### DISCUSSION TITLE
Create users without email

### DISCUSSION THREAD

**Discussion by koochoy97** [Budibase/budibase#10454] [Help] [answered]
Hello!

I'm trying to build an internal software for my company. The problem is that many employees doesn't have email.

Is there a possibility to create users only with his personal ID?

I'm using the self hosted version on digital ocean.

Thanks

**Comment by ConorWebb96** [comment, marked as answer]
Hey @koochoy97,

You need to use an email when setting up an account. However, this email doesn't need to be a real email address you can use something like testID123@test.com. Then click add user button.
<img width="483" alt="Screenshot 2023-05-02 at 09 14 12" src="https://user-images.githubusercontent.com/126772285/235614530-b12d493b-b473-41f2-925a-652b16c900e1.png">

After the above click the generate password button.
<img width="491" alt="Screenshot 2023-05-02 at 09 15 46" src="https://user-images.githubusercontent.com/126772285/235614835-f7546d87-9fdf-4e48-9a43-823556a1ae91.png">

From here you can then send them their email with the id contained within as well as the generated password you received. On login they will be prompted with changing the generated password.

I hope this helps.
